apertium_apy/handlers/translate_doc.py

```diff
@@ -56,20 +49,11 @@ async def translate_doc(file_to_translate, fmt, mode_file, unknown_marks=False):
 
 
 class TranslateDocHandler(TranslateHandler):
-    mime_type_command = None
 
     @classmethod
     def get_mime_type(cls, f):
-        if not cls.mime_type_command:
-            for command in MIMETYPE_COMMANDS.keys():
-                if shutil.which(command):
-                    cls.mime_type_command = command
-                    break
-
-        if not cls.mime_type_command:
-            return None
-        mime_command = MIMETYPE_COMMANDS[cls.mime_type_command]
-        mime_type = mime_command(f).split(';')[0]
+        mime = magic.Magic(mime = True)
+        mime_type = mime.from_file(f).split(';')[0]
```

Comment:
What if this list is empty?

Reply:
We could check if the list is empty and return None in that case?

Reply:
We could check if the list is empty and return None in that case?